Relativity.Environment.SDK.md

```diff
@@ -0,0 +1,17 @@
+# Relativity.Environment.SDK Version Compatibility
+
+[![nuget](https://img.shields.io/nuget/v/Relativity.Environment.SDK.svg)](https://www.nuget.org/packages/Relativity.Environment.SDK)

```

Comment:
This will be available in NuGet repo after pipeline run: https://trident.kcura.corp/dea/job/ExternalNugetDevelopment/job/PublishToNuGet/job/PublishToNuget/
Will do it right away after merging this PR